docs/guide/dapp-compatibility.md

```diff
@@ -5,3 +5,25 @@ Include MetaMask mobile as a listed wallet in your application, and link to open
 Please direct your users to either the listing on their preferred app store, or to [our homepage](https://metamask.io/download.html) to download MetaMask mobile.
 
 Support the MetaMask inpage provider and test that your integration works well.
+
+## MetaMask Mobile Compatibility Checklist
+
+To ensure proper compatiblilty with MetaMask Mobile, your Dapp should:
+
+- Have the ability to connect to an Ethereum Provider. ([Here](https://github.com/MetaMask/detect-provider))
+- Have an option to use MetaMask Mobile as a Wallet.
+  - Create a `Connect with MetaMask` Button.
+  - Create a [Deeplink](https://metamask.github.io/metamask-deeplinks/#) to open the Dapp in the MetaMask App.
+
+### User Doesn't have Metamask?
+- Redirect the user to the App store to download the app
+
+### Calling Transactions
+In order for MetaMask to work properly with your dapp, it's best to have clear logic that can tell the Ethereum Provider when to trigger, sign, confirm, and approve transactions. Please see our [Mobile Best Practices](/guide/mobile-best-practices) page.
```

Comment:
I'm not really sure what this section is trying to convey :thinking:

Reply:
Hmmmm I was trying to explain that your transaction flow should be clear and cut. Specifically because the MetaMask extension doesn' t work for very awkward non-clear roles. I guess it's kind of a given, know that I think about it.

Reply:
I changed up the verbiage a bit take a look at my most recent commit and let me know what you think!